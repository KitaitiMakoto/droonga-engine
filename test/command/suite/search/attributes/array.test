#@include fixture/documents.jsons
{
  "type": "search",
  "dataset": "Droonga",
  "body": {
    "queries": {
      "result": {
        "source": "Sections",
        "sortBy": ["_key"],
        "output": {
          "elements": [
            "startTime",
            "elapsedTime",
            "count",
            "attributes",
            "records"
          ],
          "format": "complex",
          "limit": 3,
          "attributes": [
            {
              "label": "section-id",
              "source": "_key"
            }
          ]
        }
      }
    }
  }
}
