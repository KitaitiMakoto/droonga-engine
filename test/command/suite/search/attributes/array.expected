{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "search.result",
  "body": {
    "result": {
      "count": 9,
      "attributes": [
        {
          "name": "section-id",
          "type": "ShortText",
          "vector": false
        }
      ],
      "records": [
        {
          "section-id": "1.1"
        },
        {
          "section-id": "1.2"
        },
        {
          "section-id": "1.3"
        }
      ]
    }
  }
}
